DWIConvert/DWIConvertLib.cxx

```diff
@@ -208,10 +208,10 @@ DWIConvert::write()
 }
 
 DWIConverter *
-DWIConvert::CreateDicomConverter(const std::string inputDicomDirectory,
-                                 const bool        useBMatrixGradientDirections,
-                                 const double      smallGradientThreshold,
-                                 const bool        allowLossyConversion)
+DWIConvert::CreateDicomConverter(const std::string & inputDicomDirectory,
+                                 const bool          useBMatrixGradientDirections,
```

Comment:
Are bools and doubles okay being passed by value?

Reply:
This is similar to the comment above for `size_t`. Bools and doubles are not expensive to copy so converting them to pass-by-reference wouldn't increase efficiency.